### PR TITLE
Add taskverse plugin

### PR DESCRIPTION
**Pull request by elioengcomp**
Add taskverse plugin.

### CODE REVIEW

File: plugins/taskverse.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# mandatory:
+pluginName: taskverse
+version: v1.0.0
+repository: https://github.com/jfrog/jfrog-cli-plugins
```
Comment (elioengcomp) [inline plugins/taskverse.yml:4]: I'm working on moving the source to its own repository.

Comment (elioengcomp) [reply plugins/taskverse.yml]: Done.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
* Will you be to upgrade the version of the `jfrog-cli-core` and `jfrog-client-go` dependencies used by the plugin code to the latest available versions?
* It is also recommended to add a `RELEASE.md` file to the plugin codebase with the release notes, and reference it from the `README.md` file. See other plugins as a reference.
* In the "About this plugin" README section, I recommend replacing *Pipelines* with *JFrog Pipelines*, for more clarity about the product this plugin integrates with.
image

Once you make this change, we'll merge tis PR, and the tests will be autimatically triggered.

**Comment by elioengcomp** [comment]
> * Will you be to upgrade the version of the `jfrog-cli-core` and `jfrog-client-go` dependencies used by the plugin code to the latest available versions?
> * It is also recommended to add a `RELEASE.md` file to the plugin codebase with the release notes, and reference it from the `README.md` file. See other plugins as a reference.
> * In the "About this plugin" README section, I recommend replacing _Pipelines_ with _JFrog Pipelines_, for more clarity about the product this plugin integrates with.
>   image

Done.

**Review by eyalbe4 — CHANGES_REQUESTED** [review]
Will you be able to remove from this PR the README change? The PR should include only the plugin descriptor file (`plugins/taskverse.yml`) in this case.
The automatic tests that make sure the plugin is ready to be plublished will fail otherwise.
This is one of the guidelibes in the [JFrog CLI Plugin Developer Guide
](https://github.com/jfrog/jfrog-cli/blob/v2/guides/jfrog-cli-plugins-developer-guide.md)
image

**Comment by elioengcomp** [comment]
> Will you be able to remove from this PR the README change? The PR should include only the plugin descriptor file (`plugins/taskverse.yml`) in this case. The automatic tests that make sure the plugin is ready to be plublished will fail otherwise. This is one of the guidelibes in the [JFrog CLI Plugin Developer Guide ](https://github.com/jfrog/jfrog-cli/blob/v2/guides/jfrog-cli-plugins-developer-guide.md) image Once you make this change, we'll merge tis PR, and the tests will be autimatically triggered.

Done.

**Comment by eyalbe4** [comment]
Congrats @elioengcomp!
The **taskverse** plugin is released.
You can install it by running `jf plugin install taskverse`.

**Comment by elioengcomp** [comment]
Awesome @eyalbe4!

I did try it and it is working like a charm.

Thank you for the guidance.

**Pull request merged**